staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -211,26 +228,40 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		return nil
 	})
 
-	if openAPIConfig != nil {
+	return s, nil
+}
+
+func (s *APIAggregator) PrepareRun() (preparedAPIAggregator, error) {
```

Comment:
shouldn't we return `*preparedAPIAggregator, error` ? Just to be consistend with `s *APIAggregator`

Reply:
no, it's just a type wrapper. A pointer does not add anything.